src/tests/Common/CLRTest.Execute.Bash.targets

```diff
@@ -444,6 +444,35 @@ $__Command $HARNESS_RUNNER apple just-run %5c
   -v
 CLRTestExitCode=$?
 
+# Exit code of xharness is zero when tests finished successfully
```

Comment:
Can we avoid duplicating the above block? I see the only difference are the two `--set-env` arguments. If they don't harm anything we could just pass the to NativeAOT runs. If they're harmful, maybe we can extract that part into a separate property?

Reply:
Yes, the Native AOT template doesn't rely on the environment arguments and can be removed.